plugins/quickinspector/quickitemgeometry.cpp

```diff
@@ -112,6 +142,10 @@ void QuickItemGeometry::initFrom(QQuickItem *item)
         QQuickItemPrivate *parentPriv = QQuickItemPrivate::get(parent);
         parentTransform = parentPriv->itemToWindowTransform();
     }
+
+    traceColor = colorForItem(item);
+    traceTypeName = ObjectDataProvider::typeName(item);
+    traceName = ObjectDataProvider::name(item);
```

Comment:
This seems to be the only dependency on probe code in the shared bit, can probably be addressed by moving this to the caller instead.